algorithms/refinement/parameterisation/detector_parameters.py

```diff
@@ -546,363 +546,11 @@ def compose(self):
         ]
 
     def get_state(self, multi_state_elt=0):
-
         # There is only one detector, but the req. panel must be specified
         panel = (self._model)[multi_state_elt]
         return matrix.sqr(panel.get_d_matrix())
 
 
-class PyDetectorParameterisationMultiPanel(DetectorParameterisationMultiPanel):
```

Comment:
I kept this around as sort of documentation, because it is much easier to understand what is going on in the Python `compose` method than the base class method, which calls out to C++
https://github.com/dials/dials/blob/afd9f7c613d373caaac62db26d56191c24d413a3/algorithms/refinement/parameterisation/detector_parameters.py#L523-L537

However, I can see that leaving bits of dead code around as "documentation" is far from ideal. What would be better? Perhaps we should keep the code for use in a test that specifically just checks that the results from the Python version match those from the one that uses the C++ function?

Reply:
having it live on in a `test_*.py` file sounds like a good idea in that case